clang/lib/Sema/SemaDecl.cpp

```diff
@@ -255,8 +255,8 @@ static ParsedType recoverFromTypeInKnownDependentBase(Sema &S,
   S.Diag(NameLoc, diag::ext_found_via_dependent_bases_lookup) << &II;
 
   ASTContext &Context = S.Context;
-  auto *NNS = NestedNameSpecifier::Create(Context, nullptr, false,
-                                          cast<Type>(Context.getRecordType(RD)));
+  auto *NNS = NestedNameSpecifier::Create(
+      Context, nullptr, false, cast<Type>(Context.getRecordType(RD)));
```

Comment:
Please, apply clang-format for diff/patch only.

Reply:
Sorry.
Done.